### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: PluginGemfile [hunk 1]
```diff
@@ -1,9 +1,13 @@
+source 'https://rubygems.org'
```
Comment (rnestler) [inline PluginGemfile:1]: Why was this removed in the first place? @schmijos knows maybe?

**Review by aneshodza — APPROVED** [review]

File: PluginGemfile (hunk 1 above)
Comment (schmijos) [reply PluginGemfile:1]: `PluginGemfile` is merged into the Redmine gemfile. Adding this line again leads to duplicate sources, doesn't it?

Comment (rnestler) [reply PluginGemfile:1]: Ah I see. Should we use a separate `Gemfile` and `Gemfile.lock`? You did delete both in https://github.com/renuo/redmine_tracky/commit/c7b74a99fabcb5af5459ba480b5edf9d9f1aa550
